content/source/docs/enterprise/users-teams-organizations/teams.html.md

```diff
@@ -4,14 +4,83 @@ page_title: "Teams - Terraform Enterprise"
 sidebar_current: "docs-enterprise2-users-teams-organizations-teams"
 ---
 
+[organizations]: ./organizations.html
+[organization settings]: ./organizations.html#organization-settings
+[users]: ./users.html
+
 # Teams
 
-Teams are groups of users that reflect your company's group
-structure. Organization owners can create teams and manage user membership.
-Organization owners can give teams read, write, and admin permissions to
-workspaces.
+Teams are groups of Terraform Enterprise (TFE) [users][] within an [organization][organizations]. To delegate provisioning work, the organization's owners can grant workspace permissions to specific teams.
+
+Teams can only have permissions on workspaces within their organization, although any user in the team can belong to teams in other organizations.
```

Comment:
I think I might say "any user in a team", since there's no specific team in question here.